spec/models/book_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+describe Book do
```

Comment:
always leave a blank line between what you require and the start of the main block